modules/Core/pages/panel/pages.php

```diff
@@ -23,6 +23,48 @@
 // Load modules + template
 Module::loadPage($user, $pages, $cache, $smarty, [$navigation, $cc_nav, $staffcp_nav], $widgets, $template);
 
+function validate_page_form() {
```

Comment:
While I like the idea of removing duplicate validation checks, I don't know if this is the most elegant way we can do it...

Reply:
Any other ideas? A collection of "form validators" in a separate directory somewhere?

Reply:
Perhaps - that seems good.
`FormValidator` class which contains a bunch of static functions seems alright. Ideally we use something like https://laravel.com/docs/9.x/validation#form-request-validation but can't get too ahead of ourselves 😉

Reply:
should that be included in this PR, or can it be done later?

Reply:
let's do that later:tm:

for now, let's keep the repeated validation in the file.